src/maxwell-web-view.c

```diff
@@ -644,6 +644,7 @@ static void
 maxwell_web_view_realize (GtkWidget *widget)
 {
   MaxwellWebViewPrivate *priv = MAXWELL_WEB_VIEW_PRIVATE (widget);
+  GString *script = g_string_new ("");
```

Comment:
You could initialize this only if `priv->script_loaded` and then check for `if (script)` below; that would make things a bit cleaner IMO. (Also true for the other cases touched by this commit.)